src/ProfiledAot/CommonMethods.cs

```diff
@@ -0,0 +1,27 @@
+// This class represents common code paths we always want to AOT
+
+// System.Threading.Tasks.Task
+// System.Net.Http.HttpClient
+// System.Console
```

Comment:
What calls System.Console in a default app?

Reply:
I could maybe remove this one.

Reply:
I'm not necessarily asking for it to be removed. More for education on "why would something in the default app call System.Console?" If we are, what is the scenario? etc.

Reply:
I saw it in the list when looking at JIT'd methods of the podcast app, but they might just have a random call somewhere.